src/components/picker/api/picker.api.json

```diff
@@ -108,13 +108,403 @@
+        "sections": [
+          {
+            "type": "section",
+            "title": "Usage Example",
+            "description": "Pickers allow selection of an option or multiple options from a list usually displayed using an actionSheet. Pickers can be used in forms, settings screens or as part of an interactive flow.",
+            "content": [
+              {
+                "value": "https://embed.figma.com/design/xFjvYNkGTmYTGYMLrmz9Ir/Guidelines-to-Docs?node-id=4415-185562&embed-host=share"
+              }
+            ]
+          },
+          {
+            "type": "list",
+            "items": [
+              {
+                "title": "Form Field (default)",
+                "description": "Mainly used as part of a form, works cohesively with the text field component and other form related components.",
+                "content": [
+                  {
+                    "value": "https://wixmp-1d257fba8470f1b562a0f5f2.wixmp.com/mads-docs-assets/assets/Components%20Docs/Picker/picker_types_form.png"
+                  }
+                ]
+              },
+              {
+                "title": "Filter Field",
+                "description": "Used mostly for filtering a screen or part of it. Can be combined with search functionality.\nThe colon symbol in the Label is not part of the component itself and should be included as part of the text string.",
+                "content": [
+                  {
+                    "value": "https://wixmp-1d257fba8470f1b562a0f5f2.wixmp.com/mads-docs-assets/assets/Components%20Docs/Picker/picker_types_filter.png"
+                  }
+                ]
+              },
+              {
+                "title": "Settings Field",
+                "description": "Based on ListItem component.\nUsed mostly when the picker is part of a settings screen.",
+                "content": [
+                  {
+                    "value": "https://wixmp-1d257fba8470f1b562a0f5f2.wixmp.com/mads-docs-assets/assets/Components%20Docs/Picker/picker_types_settings.png"
+                  }
+                ]
+              }
+            ],
+            "layout": "horizontal",
+            "title": "Picker Types",
+            "description": "Various types of pickers are available to accommodate different needs and use cases. "
+          },
+          {
+            "type": "section",
+            "title": "Picker Types Examples",
+            "content": [
+              {
+                "value": "https://embed.figma.com/design/xFjvYNkGTmYTGYMLrmz9Ir/Guidelines-to-Docs?node-id=4415-185650&embed-host=share"
+              }
+            ]
+          },
+          {
+            "type": "table",
+            "columns": ["State/Type/Preset", "Preview"],
+            "items": [
+              {
+                "title": "Default",
+                "content": [
+                  {
+                    "flexed": true,
+                    "props": {
+                      "preset": "outline",
```

Comment:
The default can't have a preset=outline... Also, we might want to add examples for the underline and outline presets?